services/frontend/src/utils/index.ts

```diff
@@ -106,12 +106,32 @@ export const attachUUID = (key: string): string => {
   return key + "-" + uuidv4();
 };
 
+export const setNodeIcon = (nodeType: string) => {
+  switch (nodeType) {
+    case "service":
+      return "ServerIcon";
+    case "volume":
+      return "DatabaseIcon";
+    case "network":
+      return "ChipIcon";
+    default:
+      return "ServerIcon";
+  }
+};
+
```

Comment:
1. You could use a map instead.
2. Why is the function called `set`, when it's just returning values?